superset-frontend/src/explore/controlPanels/BigNumberTotal.js

```diff
@@ -17,6 +17,7 @@
  * under the License.
  */
 import { t } from '@superset-ui/translation';
+import { HeaderFontSize, SubheaderFontSize } from './Shared_BigNumber';
```

Comment:
@kristw there are only a couple of shared controls between the two BigNumber viz modules, so I made this file to share them within that viz'package/repo. This is just a small example, but I was planning to follow the same example for the NVD3 and DeckGL modules, and consolidate shared controls that are relevant to only their repos.

When we get to the SUPER commonly used controls, we could follow this pattern again, and move the "global" controls (or whatever we call it) to `superset-ui`.

This also means we can still remove `controlOverrides` sectopms from the plugin control files, and do something like:
```
controlSetRows: [
        [Object.assign({},HeaderFontSize,{title: 'someNewTitle'}],
      ],
```
... if that's simpler to interpret.